modules/ui/feature_list.js

```diff
@@ -332,12 +333,26 @@ export function uiFeatureList(context) {
                 .attr('class', 'entity-type')
                 .text(function(d) { return d.type; });
 
+            label.each(function(d) {
+                if (d.entity?.type !== 'relation') return;
+                const hasColor = d.entity.tags.colour && isColourValid(d.entity.tags.colour);
+                const hasRef = d.entity.tags.ref;
+                if (hasColor || hasRef) {
+                    const color = isColourValid(d.entity.tags.colour) ? d.entity.tags.colour : '#555';
```

Comment:
Many recreational routes are associated with multiple colors, which could be tagged as a semicolon-delimited list in `colour=*` (regardless of any `osmc:symbol=*`).

Reply:
Yeah, I was pondering about `osmc:symbol` for hiking routes (but only for a future iteration on top of this feature). Multi-value `colour` tags seem to exist ([example](https://www.openstreetmap.org/relation/7840712)), but are not very frequent outside of the Netherlands:

[!<img src="https://github.com/user-attachments/assets/1de64420-d760-4172-af63-123a3db4efe1" width="360">](https://overpass-turbo.eu/s/24ld)